portal-impl/src/com/liferay/portal/language/LanguageImpl.java

```diff
@@ -1984,9 +1984,9 @@ private CompanyLocalesBag(long companyId) {
 		private final Set<Locale> _availableLocales;
 		private final Set<String> _duplicateLanguageCodes;
 		private final Map<String, Locale> _languageCodeLocalesMap =
-			new HashMap<>();
+			new LinkedHashMap<>();
```

Comment:
@Seiphon same thing here, I don't see anywhere we use _languageCodeLocalesMap's all values. I doubt this is needed.

Reply:
I sent a new pull request, so you can close this one